docs/extensions/process.md

```diff
@@ -0,0 +1,440 @@
+
+        to find existing or ongoing efforts
+        that may support what you are trying to add into the BIDS Specification.
+        Someone may have already done work for you: so avoid duplicating efforts!
+
+    -   familiarized yourself with the BIDS community by browsing current issues,
+        discussions, and proposed changes on
+        [the bids specification repository][specification_gh].
+        Search for [issues](https://github.com/bids-standard/bids-specification/issues) relating to your feature or BEP idea
+        before creating a new issue.
+
+    If you are sure that you want to move forward, also make sure you have:
+
+    -   read the [BIDS governance document](../collaboration/governance.md)
+    -   the [BIDS code of conduct](../collaboration/bids_github/CODE_OF_CONDUCT.md)
+
+### Overview of the BEP process
+
+Once a BEP has been opened (see the next [section](#starting-your-bep)),
+it goes from `Draft BEP` to `Proposed BEP` to `Merged BEP`.
+
+A `Draft BEP` is usually a Google document (see [governance](../collaboration/governance.md#draft-bep)),
+where as `Proposed BEP` refert to pull request to BIDS specification repository
+(see [governance](../collaboration/governance.md#proposed-bep)).
+
+```mermaid
+flowchart TD
+
+draft_bep[Draft BEP]
+proposed_bep[Proposed BEP]
+merged_bep[Merged BEP]
+
+draft_bep --> proposed_bep --> merged_bep
+```
+
+The transitions between those stages is decided by a review process.
+
+```mermaid
+flowchart TD
+
+draft_bep[Draft BEP]
+draft_bep_review([Draft BEP review])
+proposed_bep[Proposed BEP]
+proposed_bep_review([Proposed BEP review])
+merged_bep[Merged BEP]
+
+draft_bep --> draft_bep_review
+draft_bep_review -- negative evaluation --> draft_bep
+draft_bep_review -- positive evaluation --> proposed_bep
+proposed_bep --> proposed_bep_review
+proposed_bep_review -- negative evaluation --> proposed_bep
+proposed_bep_review -- positive evaluation --> merged_bep
+```
+
+It may also be decided to close a BEP that is not actually necessary anymore
+or is in fact beyond the scope of BIDS.
+
+```mermaid
+flowchart TD
+
+draft_bep[Draft BEP]
+draft_bep_review([Draft BEP review])
+proposed_bep[Proposed BEP]
+proposed_bep_review([Proposed BEP review])
+merged_bep[Merged BEP]
+closed_BEP[Closed BEP]
+
+draft_bep --> closed_BEP
+proposed_bep --> closed_BEP
+draft_bep --> draft_bep_review
+draft_bep_review -- negative evaluation --> draft_bep
+draft_bep_review -- positive evaluation --> proposed_bep
+proposed_bep --> proposed_bep_review
+proposed_bep_review -- negative evaluation --> proposed_bep
+proposed_bep_review -- positive evaluation --> merged_bep
+```
+
+### Deliverables
+
+In the end a BEP should produce several deliverables
+that will be evaluated during the `Proposed BEP review`.
+
+```mermaid
+flowchart TD
+
+draft_bep[Draft BEP]
+draft_bep_review([Draft BEP review])
+proposed_bep[Proposed BEP]
+PR_examples[REQUIRED
+            Pull request to examples repository]
```

Comment:
could incorporate URL hyperlink, WDYT?

```suggestion
            Pull request to <a href='https://github.com/bids-standard/bids-examples'>examples repository</a>]
```

since images, should we shorten to 

```suggestion
            PR to <a href='https://github.com/bids-standard/bids-examples'>examples repository</a>]
```

with abbreviation in the text? After all we have BEP here too anyways

Reply:
not sure that mermaid handles link well within mkdocs: let's keep it for later